configs/config.clevo_ns70pu

```diff
@@ -0,0 +1,31 @@
+CONFIG_CCACHE=y
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_NO_STAGE_CACHE=y
+CONFIG_VENDOR_CLEVO=y
+CONFIG_MAINBOARD_VERSION="darp8"
+CONFIG_CBFS_SIZE=0xA00000
+# CONFIG_CONSOLE_SERIAL is not set
+# CONFIG_POST_IO is not set
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="Darter Pro"
+# CONFIG_USE_PM_ACPI_TIMER is not set
+# CONFIG_IFD_BIN_PATH="$(FIRMWARE_OPEN_MODEL_DIR)/fd.rom"
+# CONFIG_ME_BIN_PATH="$(FIRMWARE_OPEN_MODEL_DIR)/me.rom"
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+# CONFIG_HAVE_IFD_BIN=y
+CONFIG_PCIEXP_HOTPLUG_BUSES=42
+CONFIG_PCIEXP_HOTPLUG_MEM=0xc200000
+CONFIG_PCIEXP_HOTPLUG_PREFETCH_MEM=0x1c000000
+CONFIG_COREBOOT_ROMSIZE_KB_32768=y
+CONFIG_UART_PCI_ADDR=0x0
+CONFIG_USE_INTEL_FSP_TO_CALL_COREBOOT_PUBLISH_MP_PPI=y
+# CONFIG_VALIDATE_INTEL_DESCRIPTOR=y
+# CONFIG_HAVE_ME_BIN=y
+CONFIG_PCIEXP_HOTPLUG_IO=0x2000
+CONFIG_SUBSYSTEM_VENDOR_ID=0x0000
+CONFIG_SUBSYSTEM_DEVICE_ID=0x0000
+CONFIG_I2C_TRANSFER_TIMEOUT_US=500000
+CONFIG_SMMSTORE=y
+CONFIG_SMMSTORE_V2=y
+CONFIG_SMMSTORE_SIZE=0x40000
+# CONFIG_PAYLOAD_ELF=y
+# CONFIG_PAYLOAD_FILE="$(FIRMWARE_OPEN_UEFIPAYLOAD)"
```

Comment:
Can be removed. In general, the defconfig should be generated using `make menuconfig`, making your changes, and `make savedefconfig` instead of editing it manually.

Reply:
Removed.